lib/split/trial.rb

```diff
@@ -70,19 +71,23 @@ def choose!(context = nil)
         else
           self.alternative = @user[@experiment.key]
           if alternative.nil?
-            self.alternative = @experiment.next_alternative
+            if @experiment.cohorting_disabled?
```

Comment:
Every time `cohorting_disabled?` is called we trigger a new call to redis.

I'd like to keep redis calls to a minimum, so for now, we can memoize the value on experiment, or call it only once on this method.

Reply:
👍